keyboards/skeletonkbd/frost68/keymaps/default/keymap.c

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 skeletonkbd (@skeletonkbd)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names { _BASE, _L1, _L2, _L3 };
+
+// clang-format off
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[_BASE] = LAYOUT_alice(
+		KC_ESC,  KC_1,    KC_2,    KC_3,    KC_4,     KC_5,   KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,  KC_BSPC,          KC_DEL,
+		KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,     KC_T,            KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC, KC_BSLS, KC_HOME,
+		KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,     KC_G,            KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,           KC_END,
+		KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,     KC_B,            KC_LNG1, KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_RSFT, KC_UP,
+		KC_LCTL, KC_LALT,          KC_SPC,            MO(1),           KC_SPC,                    KC_RGUI,          KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT),
```

Comment:
Since you made an alias for the layer, let's use it....
```suggestion
		KC_LCTL, KC_LALT,          KC_SPC,            MO(_L1),         KC_SPC,                    KC_RGUI,          KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT),
```